pkg/sdn/plugin/vnids_master.go

```diff
@@ -0,0 +1,324 @@
+package plugin
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	log "github.com/golang/glog"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	utilruntime "k8s.io/kubernetes/pkg/util/runtime"
+	"k8s.io/kubernetes/pkg/util/sets"
+	utilwait "k8s.io/kubernetes/pkg/util/wait"
+	"k8s.io/kubernetes/pkg/watch"
+
+	osapi "github.com/openshift/origin/pkg/sdn/api"
+	pnetid "github.com/openshift/origin/pkg/sdn/plugin/netid"
+)
+
+type masterVNIDMap struct {
+	// Synchronizes assign, revoke and update VNID
+	lock         sync.Mutex
+	ids          map[string]uint32
+	netIDManager *pnetid.Allocator
+
+	adminNamespaces sets.String
+}
+
+func newMasterVNIDMap() (*masterVNIDMap, error) {
+	netIDRange, err := pnetid.NewNetIDRange(osapi.MinVNID, osapi.MaxVNID-osapi.MinVNID+1)
+	if err != nil {
+		return nil, fmt.Errorf("unable to create NetID range: %v", err)
+	}
+
+	return &masterVNIDMap{
+		netIDManager:    pnetid.NewInMemory(netIDRange),
+		adminNamespaces: sets.NewString(kapi.NamespaceDefault),
+		ids:             make(map[string]uint32),
+	}, nil
+}
+
+func (vmap *masterVNIDMap) GetVNID(name string) (uint32, bool) {
```

Comment:
The masterVNIDMap functions are inconsistent about initial-uppercase vs initial-lowercase. It looks like none of them get called from outside vnids_master.go (unlike the corresponding nodeVNIDMap functions), so maybe make them all lowercase?